romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -91,6 +92,12 @@ def create_image_model(input_model, image_info):
             }
     out_node = rds.WfiImage(inst)
     im = rdd.ImageModel(out_node)
+
+    # trim off border reference pixels from science data, dq, err
+    im.data = im.data[:, 4:-4, 4:-4]
+    im.dq = im.dq[:, 4:-4, 4:-4]
+    im.err = im.err[:, 4:-4, 4:-4]
```

Comment:
These lines need testing.

Reply:
I added tests for these but codecov still says the lines aren't covered. Could it be because I used pytest skip if CI? the test i wrote runs locally

Reply:
The CI skip is only for runs external to the institute. The roman test CRDS server is only available on vpn so it will fail for the CI runs that are not on VPN.

Reply:
hmm ok, i still dont understand why these lines are marked as uncovered then

Reply:
Did you commit the test file? This ticket only has 4 files attached to it at present:
CHANGES.rst
romancal/dq_init/dq_init_step.py 
romancal/dq_init/tests/test_dq_init.py 
romancal/ramp_fitting/ramp_fit_step.py 
There is no ramp test file added here.

Reply:
the test i added is in test_dq_init.py that should cover the untested lines in dq_init_step.py. I'm going to add the coverage to ramp in a separate PR since the existing test (which i wanted to just add assert statements to) is marked as xfail and needs to be fixed.

Reply:
The test files are created on-the-fly, the generate_wfi_reffiles needs to be updated.